docs/publishing/submit-your-plugin.md

```diff
@@ -24,7 +24,7 @@ In this step, you'll prepare a release for your plugin that's ready to be submit
 
 1. In `manifest.json`, update `version` to a new version that follows the [Semantic Versioning](https://semver.org/) specification.
 
-1. [Create a GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release). The name of the release must match the version in your `manifest.json`. Don't include a `v` in the release name.
+1. [Create a GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release). The  "Tag version" of the release must match the version in your `manifest.json`. Don't include a `v` in the tag version. It is also good practice to provide a release title and to describe the release in the description field. 
```

Comment:
```suggestion
1. [Create a GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release). 
   - The  "Tag version" of the release must match the version in your `manifest.json`. 
   - Don't include a `v` in the tag version. 
1. Enter a name for the release, and describe it in the description field.
```

Good catch!

- If it's a good practice, we should ask the reader to do it.
- For text fields, the Microsoft style guide uses "Enter" rather than "Provide".